db/storage.go

```diff
@@ -0,0 +1,94 @@
+package db
+
+import (
+    "database/sql"
+    _ "github.com/go-sql-driver/mysql"
+    "strconv"
+
+    "./../error"
+    "./../config"
+)
+
+const configName = "db"
+
+type storageEntity struct {
+    db *sql.DB
+    stmt []*(sql.Stmt)
+    rows []*(sql.Rows)
+}
+
+var Storage *storageEntity
+
+func NewStorage(config *config.Config) (*storageEntity) {
```

Comment:
Your config has an interface, `GetDb(), GetEmail()...`. Try to return a struct and accept an interface, if possible. Replace `config *config.Config` to something like `config.Config` and make struct `config.Config` private (start struct name from lowercase letter)